### PR TITLE
Address codeQL issues

### PR DESCRIPTION
**Pull request by jlevin**

### CODE REVIEW

File: vg/test_helpers.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 def get_imported_names(module):
-    return [name for name in dir(module) if not name.startswith("_")]
+    return sorted([name for name in module.__all__ if not name.startswith("_")])
```
Comment (paulmelnikow) [inline vg/test_helpers.py:2]: Is anything with underscores being exported?

Comment (jlevin) [reply vg/test_helpers.py:2]: No

Comment (paulmelnikow) [reply vg/test_helpers.py:2]: Then this is equivalent to `sorted(module.__all__)`

Comment (jlevin) [reply vg/test_helpers.py:2]: updated in https://github.com/lace/vg/pull/232

Comment (paulmelnikow) [reply vg/test_helpers.py:2]: thanks!

**Pull request merged**